source/usage-examples/deleteOne.txt

```diff
@@ -71,4 +71,4 @@ If you run the example above, you should see output that resembles the following
 .. code-block:: none
    :copyable: false
 
-   'Successfully deleted one document.'
+   Successfully deleted one document.
```

Comment:
Please mention what happens on subsequent runs of the program [as done here](https://github.com/mongodb/docs-node/pull/238/files#diff-014ee4e2fab0536984f9cf7f3c28da175549f3c28f0e0da8d2cf6fb972a87f96R63-R68)